packages/components/src/images/trezor_logo_suite.svg

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
is this doctype tag in svg necessary? (just curious, i didnt see that in other svg files and doesnt have a close tag)

Reply:
probably we can get rid of that, I'll run it through SVG optimizer. Thanks